yarn.lock

```diff
@@ -4,14 +4,14 @@
 
 "@cspotcode/source-map-support@^0.8.0":
   version "0.8.1"
-  resolved "https://registry.yarnpkg.com/@cspotcode/source-map-support/-/source-map-support-0.8.1.tgz#00629c35a688e05a88b1cda684fb9d5e73f000a1"
+  resolved "https://registry.npmjs.org/@cspotcode/source-map-support/-/source-map-support-0.8.1.tgz"
```

Comment:
We should revert these changes back to the yarn mirror

Reply:
https://github.com/chainpatrol/discord-bot/pull/23/commits/9e893e57519b8b20c2de90484c8e00bc12fb269b